tailwind.js

```diff
@@ -138,72 +159,68 @@ module.exports = {
   },
 
   textSizes: {
-    '1': '.625rem',     // 10px
-    '2': '.75rem',     // 12px
-    '3': '.875rem',    // 14px
-    '4': '1rem',     // 16px
-    '5': '1.125rem',   // 18px
-    '6': '1.25rem',    // 20px
+    '1': '.625rem',   // 10px
+    'base': '1rem',   // 16px
+    '2': '.75rem',    // 12px
+    '3': '.875rem',   // 14px
+    '4': '1rem',      // 16px
+    '5': '1.125rem',  // 18px
+    '6': '1.25rem',   // 20px
     '7': '1.5rem',    // 24px
     '8': '2.125rem',  // 34px
-    '9': '2.75rem',      // 44px
-    '10': '4rem',      // 64px
+    '9': '2.75rem',   // 44px
   },
 
   fontWeights: {
     'thin': 100,
-    'ultralight': 200,
-    'light': 300,
     'normal': 400,
-    'medium': 500,
-    'semibold': 600,
     'bold': 700,
-    'ultrabold': 800,
-    'black': 900,
   },
 
   leading: {
-    '0': 1,
-    '1': 1.15,
-    '2': 1.25,
-    '3': 1.5,
-    '4': 1.75,
+    'none': 1,
+    'tight': 1.25,
+    'normal': 1.5,
+    'loose': 2,
```

Comment:
My preference for leading and tracking is to use more descriptive terms, but I can see an argument for using a difference scale.